ts/smelter/src/api.generated.ts

```diff
@@ -6,8 +6,7 @@
  */
 
```

Comment:
rebase to master and regenerate this before merging, those changes are already on master